introduction_to_amazon_algorithms/forecasting_services_comparison/forecast_example.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    import util
</pre><p>CI is failing at this step - I don't see a util.py file in this PR - you would need to add it so it can be imported. However, I think it's a better option to attach the <code>AmazonForecastFullAccess</code> policy to the current SageMaker execution role, instead of creating a new role, since more customers can modify roles than create them. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3532/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>